Snakefile

```diff
@@ -720,7 +720,7 @@ rule monolingual_subsample_en:
     output:
         "data/monolingual/en.sample.txt"
     shell:
-        "zcat {input} | fold -w 1000 -s | awk 'NR % 10 == 0' > {output}"
+        "zcat {input} | fold -w 1000 -s | awk 'NR % 100 == 0' > {output}"
 
```

Comment:
The comment at line 712 above should be updated to reflect the new percentage (1% rather than 10%).